clients/admin-ui/cypress/e2e/datasets.cy.ts

```diff
@@ -584,3 +587,28 @@ describe("Dataset", () => {
     });
   });
 });
+
+describe("Dataset Plus Enabled", () => {
```

Comment:
I should have mentioned [datasets-classify.cy.ts](https://github.com/ethyca/fides/blob/1aaab78a56c683921cad0f26de9693aebc2c7484/clients/admin-ui/cypress/e2e/datasets-classify.cy.ts#L13-L14) is where we've been putting all the tests for when plus is enabled.